lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -259,73 +259,115 @@ def _get_converged_core_config(service_mconfig: object) -> bool:
 
 def _get_default_slice_service_type_config(service_mconfig: object) -> str:
     """
-    Retrieves default_slice_service_type config value,If it does not exist
+    Retrieve default_slice_service_type config value,If it does not exist
     it defaults to 1.
     Args:
         service_mconfig:
 
     Returns: slice service type value
-    """
+    """  # noqa: DAR101, DAR201
+    enable_default_service_slice_type = get_service_config_value(
+        'mme', 'default_slice_service_type', None,
+    )
+
+    if enable_default_service_slice_type is not None:
+        return enable_default_service_slice_type
+
```

Comment:
this is not needed as line 276 already takes care of returning a default value.

Reply:
okay I guess you still want this to be configurable if it is defined in mme.yml file.

Reply:
yes, it's required to fetch the configurations from mme.yml, hence code added.